internal/provider/provider.go

```diff
@@ -29,86 +55,87 @@ func New(version string) func() *schema.Provider {
 					Type:        schema.TypeString,
 					Optional:    true,
 					DefaultFunc: schema.EnvDefaultFunc("CLOUDFLARE_API_USER_SERVICE_KEY", nil),
-					Description: "A special Cloudflare API key good for a restricted set of endpoints",
+					Description: "A special Cloudflare API key good for a restricted set of endpoints. Alternatively, can be configured using the `CLOUDFLARE_API_USER_SERVICE_KEY` environment variable.",
 				},
 
 				"rps": {
 					Type:        schema.TypeInt,
 					Optional:    true,
 					DefaultFunc: schema.EnvDefaultFunc("CLOUDFLARE_RPS", 4),
-					Description: "RPS limit to apply when making calls to the API",
+					Description: "RPS limit to apply when making calls to the API. Alternatively, can be configured using the `CLOUDFLARE_RPS` environment variable.",
 				},
 
 				"retries": {
 					Type:        schema.TypeInt,
 					Optional:    true,
 					DefaultFunc: schema.EnvDefaultFunc("CLOUDFLARE_RETRIES", 3),
-					Description: "Maximum number of retries to perform when an API request fails",
+					Description: "Maximum number of retries to perform when an API request fails. Alternatively, can be configured using the `CLOUDFLARE_RETRIES` environment variable.",
 				},
 
 				"min_backoff": {
 					Type:        schema.TypeInt,
 					Optional:    true,
 					DefaultFunc: schema.EnvDefaultFunc("CLOUDFLARE_MIN_BACKOFF", 1),
-					Description: "Minimum backoff period in seconds after failed API calls",
+					Description: "Minimum backoff period in seconds after failed API calls. Alternatively, can be configured using the `CLOUDFLARE_MIN_BACKOFF` environment variable.",
 				},
 
 				"max_backoff": {
 					Type:        schema.TypeInt,
 					Optional:    true,
 					DefaultFunc: schema.EnvDefaultFunc("CLOUDFLARE_MAX_BACKOFF", 30),
-					Description: "Maximum backoff period in seconds after failed API calls",
+					Description: "Maximum backoff period in seconds after failed API calls. Alternatively, can be configured using the `CLOUDFLARE_MAX_BACKOFF` environment variable.",
 				},
 
 				"api_client_logging": {
 					Type:        schema.TypeBool,
 					Optional:    true,
 					DefaultFunc: schema.EnvDefaultFunc("CLOUDFLARE_API_CLIENT_LOGGING", false),
-					Description: "Whether to print logs from the API client (using the default log library logger)",
+					Description: "Whether to print logs from the API client (using the default log library logger). Alternatively, can be configured using the `CLOUDFLARE_API_CLIENT_LOGGING` environment variable.",
 				},
 
 				"account_id": {
 					Type:        schema.TypeString,
 					Optional:    true,
 					DefaultFunc: schema.EnvDefaultFunc("CLOUDFLARE_ACCOUNT_ID", nil),
-					Description: "Configure API client to always use that account",
+					Description: "Configure API client to always use a specific account. Alternatively, can be configured using the `CLOUDFLARE_ACCOUNT_ID` environment variable.",
+					Deprecated:  "Use resource specific `account_id` attributes instead.",
```

Comment:
Hi, 

Some of the resources don't accept `zone_id` or `account_id` (i.e. [worker_script](https://github.com/cloudflare/terraform-provider-cloudflare/blob/master/internal/provider/schema_cloudflare_worker_script.go#L58)). This appears to mean we need to keep the `account_id` defined as input at the cloudflare provider level, even though it warns as Deprecated. Am I understanding this correctly? Thanks!